code/extensions/TimezoneMemberExtension.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Class TimezoneMemberExtension.
+ *
+ * Adds all available timezones as an optional field to SilverStripe {@link Member}.
+ */
+class TimezoneMemberExtension extends DataExtension
+{
```

Comment:
braces should be on the same line as the `class` declaration (at least until we change the formatting of this module to PSR-2)